doc/man3/OSSL_PARAM.pod

```diff
@@ -86,6 +101,9 @@ parameter data.
 In case the C<data_size> is too small for the data, the I<responder>
 must still set this field to indicate the minimum data size required.
 
+When the C<OSSL_PARAM> is used as a parameter descriptor,
```

Comment:
Agreed but it is also ignored when passing parameters to something.

Reply:
Sure, I can add a blurb on that